samples/servicecontrol/events-subscription/sample.md

```diff
@@ -12,20 +12,21 @@ related:
  - samples/servicecontrol/monitoring3rdparty
 ---
 
-This sample shows how to monitor heartbeat and failed message events in ServiceControl.
-
-## Prerequisites
-
- 1. [Install ServiceControl](/servicecontrol/installation.md).
- 1. Using [ServiceControl Management](/servicecontrol/license.md#servicecontrol-management-app) tool, set up ServiceControl to monitor endpoints using MSMQ transport.
- 1. Ensure the `ServiceControl` process is running before running the sample. 
+This sample shows how to monitor heartbeat and failed message events in ServiceControl, as well as observing the same activity in ServicePulse. The sample uses the [Learning Transport](/transports/learning/) and a portable version of the Particular Service Platform tools. Installing ServiceControl is **not** required.
 
 NOTE: When using the [Azure Service Bus transport](/transports/azure-service-bus/legacy/), the subscribing endpoint must also use the same name shortening strategy as ServiceControl. See the [configuration settings](/transports/azure-service-bus/configuration.md#entity-creation), or if using the [legacy Azure Service Bus transport](), see its [sanitization strategy documentation](/transports/azure-service-bus/legacy/sanitization.md)
 
-NOTE: The ServiceControl endpoint queue name configured in `EndpointsMonitor` must match the name of the ServiceControl instance with the underlying transport naming rules.
+downloadbutton
+
 
 ## Running the project
 
+Running the project will result in 3 console windows:
+
+1. **NServiceBusEndpoint**: The endpoint that represents the system being monitored.
+1. **EndpointsMonitor**: The endpoint that subscribes to ServiceControl heartbeat and failed message events.
+1. **PlatformLauncher**: Runs an in-process version of ServiceControl and ServicePulse. When the ServiceControl instance is ready, a browser window will be launched displaying the ServicePulse Dashboard.
```

Comment:
Is the dashboard the right entry point or should we point it to the failed message view?

Reply:
I played around with the failed messages view. It looks so boring before you have any failed messages that I decided the dashboard was the better way to go.